BUILD.md

```diff
@@ -54,7 +54,7 @@
     * `make install -j4`
 
 16. Go to **openssl** folder
-    * `cd /home/openssl-OpenSSL_1_1_1`
+    * `cd /home/openssl-openssl-3.0.11`
```

Comment:
Is this working? I am asking this because the ZIP that we are downloading don't have the version in the filename (and it shouldn't)

Reply:
The .zip is called `"openssl.zip"`, but it has a folder inside it that is called `"openssl-openssl-3.0.11"`, which is what we get when we unzip it